templates/home.html

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+  <title>PicForEating</title>
+
+  <script type="text/javascript" src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+  <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.2.1.min.js"></script>  
```

Comment:
Not 100% sure what it is used for - thought it was related to API to be used. Was in my notes somewhere...